modules/programs/ssh.nix

```diff
@@ -10,6 +10,35 @@ let
 
   unwords = builtins.concatStringsSep " ";
 
+  localForwardModule = types.submodule ({ ... }: {
+    options = {
+      bindAddress = mkOption {
```

Comment:
Would it make sense to change `bindAddress`, `port`, `host`, and `hostPort` to `bind.address`, `bind.port`, `host.address`, and `host.port`, respectively?